src/contractTest/java/uk/gov/hmcts/reform/adoption/CaseDocumentClientPactTest.java

```diff
@@ -12,11 +12,11 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
```

Comment:
@jamesrferguson1 Mockito was upgraded in the build.gradle, then the extensions below ere changed

Reply:
Hi @ptrelease,
Was this comment for James' information only?